distribution/std-lib/Standard/src/Image/Data/Image.enso

```diff
@@ -34,52 +37,131 @@ type Image
                 arr = Java_Image.get this.opencv_mat row column
                 Vector.Vector arr
 
-    ## Calculates the per-element sum of an image and a scalar.
+    ## Calculates the per-element sum of an image and a scalar or a matrix.
+       Matrix should have the same dimensions as the image.
+
+       Arguments:
+       - `value`: A value can be a number, a vector of numbers or a matrix.
+         Number value is applied to each pixel's channel of the image.
+         Vector value is padded with zeros to match the number of channels, and then applied to
+         each pixel of the image.
```

Comment:
These sentences don't make sense to me. You may need to clarify what you mean. You're also dropping articles like "the", "a", "an", from many of these docblocks. Please make sure you don't!